frontend/website/pages/hardware-wallet.mdx

```diff
@@ -19,9 +19,81 @@ The following debian packages are needed to use hardware wallets on linux:
 sudo apt install libusb-1.0-0-dev libudev1 libudev-dev
 ```
 
+### Make sure your computer is recognising the Ledger device
+
+- Plug in the Ledger device, unlock it with your passcode, and go to the menu. Don't open any app,
+- Run `lsusb`, and see if the device is listed -- it will have ID `2c97:0001`
+- If the Ledger device is not listed, follow the suggestions [here](https://support.ledger.com/hc/en-us/articles/115005165269-Fix-connection-issues).
+
+### Build/download the Coda Ledger application
+
+Either download the precompiled app binary from [here](...).
```

Comment:
Link?

Reply:
i didn't know where to put it, in the coda-ledger-app repo too?

Reply:
Yeah maybe in the github releases section of that repo?